src/fairseq2/models/wav2vec2/masker.py

```diff
@@ -149,8 +149,11 @@ def extract_masked_elements(seqs: Tensor, temporal_mask: Tensor) -> Tensor:
         The temporal mask. *Shape:* :math:`(N,S)`, where :math:`N` is the batch
         size and :math`S` is the sequence length.
     """
+    
+    N=seqs.size(0)
```

Comment:
Nit: do you mind naming this variable `batch_size`? I know it is more verbose, but, as a convention, we use that variable name in several places in the code base.

Reply:
Done!